test/test_cuda_multigpu.py

```diff
@@ -1121,6 +1121,61 @@ def test_streaming_backwards_device_transfer(self):
         self.assertTrue(a.grad.sum().item() == 4 * size)
         self.assertTrue(b.grad.sum().item() == 4 * size)
 
+    @unittest.skipIf(not TEST_MULTIGPU, "only one GPU detected")
```

Comment:
`TEST_MULTIGPU` indicates that this test requires at least 2 GPUs. However, this specific test case actually requires 3 GPUs, which creates a discrepancy.